### PR TITLE
#3299 Update Schema

### LINKED ISSUE #3299: Vaccine schema update
## Is your feature request related to a problem? Please describe.

Following updates to the schema:

`TemperatureBreach`
- + `TemperatureBreach.minThresholdTemperature`
- + `TemperatureBreach.maxThresholdTemperature`
- + `TemperatureBreach.thresholdDuration`
- + `TemperatureBreach.type`
- - `TemperatureBreach.temperatueBreachConfig`

`TemperatureLog`
- + `TemperatureLog.logInterval`

`TemperatureBreachConfiguration`
- + `TemperatureBreachConfiguration.type`

## Describe the solution you'd like

AS above

## Implementation

As above

## Describe alternatives you've considered

N/A

## Additional context

N/A

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3299 

## Change summary

- Updates schema as per the issue
- Updates sync with updating schema

## Testing

*Currently not testable as desktop needs an updated schema.. :)*

- [ ] Can sync with desktop

### Related areas to think about

N/A

### CODE REVIEW

File: src/database/DataTypes/TemperatureBreach.js [hunk 1]
```diff
@@ -93,7 +93,11 @@ TemperatureBreach.schema = {
     startTimestamp: { type: 'date', optional: true },
     endTimestamp: { type: 'date', optional: true },
     location: { type: 'Location', optional: true },
-    temperatureBreachConfiguration: { type: 'TemperatureBreachConfiguration', optional: false },
+    thresholdMaxTemperature: { type: 'double' },
+    thresholdMinTemperature: { type: 'double' },
+    thresholdDuration: { type: 'double' },
+    acknowledged: { type: 'boolean', default: false },
```
Comment (katherine-sussol) [inline src/database/DataTypes/TemperatureBreach.js:99]:
```suggestion
    acknowledged: { type: 'bool', default: false },
```

Was getting this in the metro bundler but it went away with the above: 
Screen Shot 2020-12-22 at 11 58 55 AM

**Review by katherine-sussol — CHANGES_REQUESTED** [review]
Cool 😎 , looks good aside from the minor hiccup

File: src/database/DataTypes/TemperatureBreach.js (hunk 1 above)
Comment (joshxg) [reply src/database/DataTypes/TemperatureBreach.js]: b1a87e9

woooooooops

**Review by katherine-sussol — APPROVED** [review]
Dog LGTM

**Pull request merged**